### PR TITLE
Add transction codec definition

### PR DESCRIPTION
**Pull request by holygits**
Had been meaning to document this

### CODE REVIEW

File: CENNZnet-API/Transaction-Codec.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+CENNZnet extrinsics are SCALE encoded payloads of the format
+```
+<payload length><version header><address><signature><options><method>
+```
+SCALE is a non-self describing binary codec used for substrate based blockchains.  
+Therefore the type to be encoded or decoded must be known by the producer or consumer ahead of time.
+This makes the encoding more terse at the expense of dynamic comprehension.
+
+Details on the [SCALE codec](https://substrate.dev/docs/en/knowledgebase/advanced/).
```
Comment (Amy-Centrality) [inline CENNZnet-API/Transaction-Codec.md:11]: hmm this link might be broken? it shows 404 when I visited..

**Review by Amy-Centrality — CHANGES_REQUESTED** [review]

File: CENNZnet-API/Transaction-Codec.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
+# Extrinsic Codec Definition
+
+CENNZnet extrinsics are SCALE encoded payloads of the format
```
Comment (Amy-Centrality) [inline CENNZnet-API/Transaction-Codec.md:3]: This seems very cool but could you please explain how this is used in the API?

Comment (holygits) [reply CENNZnet-API/Transaction-Codec.md]: Moved it to "CENNZnet developer section"

**Review by KarishmaBothara — COMMENTED** [review]
lgtm 
compared it with https://centralitydev.atlassian.net/wiki/spaces/HOW/pages/1197768705/Debugging+encode+decode+with+cargo+test

**Review by KarishmaBothara — APPROVED** [review]

**Review by Amy-Centrality — APPROVED** [review]

**Pull request merged**